packages/account-v2/src/modules/POI/POI.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useReducer } from 'react';
+import { useKycAuthStatus } from '@deriv/api-v2';
+import { Loader } from '@deriv-com/ui';
+import { AUTH_STATUS_CODES, POI_SUBMISSION_STATUS } from '../../constants';
+import { POICountrySelector, POIFlowContainer, VerificationStatus } from '../../containers';
+import { TPOIActions } from '../../utils';
+
+export const ProofOfIdentity = () => {
+    const { isLoading, kyc_auth_status: kycAuthStatus } = useKycAuthStatus();
+
+    const poiStatus = kycAuthStatus?.identity.status;
+    const service = kycAuthStatus?.identity.service;
+    const isPOARequired = kycAuthStatus?.address.status === AUTH_STATUS_CODES.NONE;
+
+    const initialState = {
+        selectedCountry: '',
+        submissionStatus: '',
+    };
+
+    const reducer = (state: typeof initialState, action: TPOIActions) => {
```

Comment:
the state here is pretty much simple, isn't it?
maybe just useReducer here is just not needed?
wouldn't it be shorter just to use useState?

Reply:
I created a reducer because I thought there might be scenarios where we need to pass state update functions to Child props. So I felt its better to use reducer as only one state update function will be passed, but can update different states based on the action